kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java

```diff
@@ -51,22 +53,21 @@ public void createTopic() {
                 .sendData()
                 .waitUntilScreenReady();
         pages.open()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS)
-                .topicIsVisible(topicToCreate.getName());
+                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
+        Assertions.assertTrue(pages.topicsList.isTopicVisible(topicToCreate.getName()));
```

Comment:
we need to leave assertion message in every place we use assertions

Reply:
Fixed